crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -174,6 +208,7 @@ async fn surplus_protocol_fee_sell_order_not_capped() {
             },
         },
         expected_score: 20.ether().into_wei(),
+        fee_handler: FeeHandler::Driver,
```

Comment:
nit: If we need to make this change in 33 lines (just so that in one case you can use a different value), maybe the abstraction is not the right use. Other options would be

1. Have a default implementation of `TestCase` which in most cases can just be expanded using ..Default::default()
2. Accept a function that allows to adjust the `Setup` before the test is executed
3. Use a builder pattern for `TestCase`

Reply:
@fleupold I couldn't agree more. I believe the builder is the best way, but I think we should address it in another PR since this one was getting already too big.